README.md

```diff
@@ -130,6 +130,14 @@ import { getPrinters } from "pdf-to-printer";
 getPrinters().then(console.log);
 ```
 
+Get printers paper names list
+
+```javascript
+import { getPrinters } from "pdf-to-printer";
+
+getPrinters(["printerPaperNames"]).then(console.log);
```

Comment:
of course, I have declared a const with default properties, we can add `printerPaperNames` for sure

```
```

Reply:
I got your point, but I have implemented a validation to not set printer data if value is empty, in some cases the Printer object will be just `deviceId` and `name`, should I set always the `printerPaperNames` property even if its empty?

https://github.com/artiebits/pdf-to-printer/blob/6d99f8908bf85c757caad005382b7c4aa05c537d/src/utils/windows-printer-valid.ts#L31-L39

following this, my type Printer always have `deviceId`, `name` and `printerPaperNames` instead of have only valid values, for example, some printers dont have any paper names, isn't preferrable to not create the `printerPaperNames` at all instead of create it empty? 

Now it will not create due the Partial<> type:

https://github.com/artiebits/pdf-to-printer/blob/6d99f8908bf85c757caad005382b7c4aa05c537d/src/index.ts#L6-L8

anyways, it's just my opinion, if you want the type Printer always have the `printerPaperNames` I will do.

Reply:
what do you think about always returning `deviceId`, `name` and `paperSizes`, but setting them to null if info is no  not available?

I also would recommend to call the new property `paperSizes`. This approach would be more consistent with the `name` property since it is not `printerName`. And the print function accepts `paperSize` as an option, so the `paperSizes` property would be more consistent with that usage as well.

Reply:
actually I prefer the `paperSizes` will be an empty array when it has no value, instead a null, what do you think?

Reply:
> actually I prefer the `paperSizes` will be an empty array when it has no value, instead a null, what do you think?

good idea, Gustavo. let's make it an empty array